vault/data_source_aws_static_credentials_test.go

```diff
@@ -0,0 +1,59 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccDataSourceAWSStaticCredentials(t *testing.T) {
+	_, _ = testutil.GetTestAWSCreds(t)
+	username := testutil.SkipTestEnvUnset(t, "AWS_STATIC_USER")[0]
+	mount := acctest.RandomWithPrefix("tf-aws-static")
+	resourceName := "data.vault_aws_static_access_credentials.creds"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion114)
+		},
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAWSStaticDataSourceConfig(mount, username),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, consts.FieldAccessKey),
+					resource.TestCheckResourceAttrSet(resourceName, consts.FieldSecretKey),
+				),
+			},
+			//testutil.GetImportTestStep("vault_aws_static_access_credentials.creds", false, nil),
```

Comment:
Can be removed, data sources are essentially imports in themselves so they don't need this step 😄